src/data/full-quiz.js

```diff
@@ -1,4 +1,22 @@
 const fullQuiz = [
+  {
+    Question: "What is the function of resize2fs command?",
+    Answer: "This can be used to change the size of an ext2 or ext3 filesystem.",
+    Distractor1: "This is used manipulate the size of partitions.",
+    Distractor2: "This is used to create an ext2, ext3, or ext4 filesystem",
+    Distractor3: "This is used to check the family of filesystems.",
+    Explanation: "The resize2fs program will resize ext2, ext3, or ext4 file systems. It can be used to enlarge or shrink an unmounted file system located on device. The resize2fs program does not manipulate the size of partitions.",

```

Comment:
I think it would sound better to add the word `the` before device.

Proposed fix:
It can be used to enlarge or shrink an unmounted file system located on the device.